site/content/en/docs/concepts/_index.md

```diff
@@ -57,16 +57,58 @@ network throughput between the Pods.
+
+<h4> Example: Provisioning AdmissionCheck </h4>
+
+Without AdmissionChecks or [TopologyAwareScheduling](docs/concepts/topology_aware_scheduling/), Kueue admissions were mainly based on quota checks - if sufficient quota existed, the workload was admitted. While quota reservation ensures logical resource availability, it doesn't guarantee physical resources exist to schedule all pods successfully. The [ProvisioningRequest AdmissionCheck](/docs/admission-check-controllers/provisioning/) addresses this in cloud environments.
+
+Kueue's enhanced admission requires two sequential checks:
+
+- Quota Reservation: Kueue validates the resource requests against ClusterQueue's available quota and resource flavors, reserves the required resources if available and locks the quota to prevent other workloads from claiming it. This step verifies logical resource availability. <br>
+- Capacity Guarantee: This step uses ProvisioningRequest and [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) to verify physical resource availability. 
+  - The Kueue controller creates a ProvisioningRequest object by attaching the Workload's PodTemplates(optionally merged via [PodSetMergePolicy](/docs/admission-check-controllers/provisioning/#podset-merge-policy)) , applying [ProvisioningRequestConfig](/docs/admission-check-controllers/provisioning/#provisioningrequest-configuration) settings, and setting owner reference to Workload.
+  - Cluster Autoscaler receives ProvisioningRequest, checks actual cluster capacity, triggers scaling if needed and updates ProvisioningRequest status with this possible states: 
+    - `Provisioned=true`: Capacity guaranteed
+    - `Provisioned=false`: Scaling in progress (Kueue surfaces ETA via Kubernetes events)
+    - `Failed=true`: Cannot provision
+    - `BookingExpired=true`: Treated same as `Failed=true`  
+    - `CapacityRevoked=true`: Evicts workloads with retry support 
+
+Let's understand this with a real-world usage - GPU Workload:
+
+Scenario: *AI training job requiring 16 GPUs*
+
+- Step 1 (Quota Reservation): ClusterQueue has 32 GPU quota available. Kueue reserves 16 GPUs from this quota.
+
+- Step 2 (Admission Check): Kueue creates a ProvisioningRequest requesting for 16 GPUs. 
+  - Cluster Autoscaler checks cloud provider GPU inventory and initiates scaling of 4x GPU nodes (4 GPUs each). It sets `Provisioned=true` when nodes are ready.
+
+  - Kueue sees the `Provisioned=true` proceeds to mark the AdmissionCheck `Ready` and admits workload
+
+Outcome:
+*Job starts immediately with all 16 GPUs available.*
+
+<h4> Failure Handling: </h4> 
+
+- If the admission check fails due to temporary issues (e.g., cloud capacity shortages), the system releases the reserved quota, requeues the workload, and triggers exponential backoff retries.
```

Comment:
Can we reference `retryStrategy` here? https://kueue.sigs.k8s.io/docs/admission-check-controllers/provisioning/#provisioningrequestconfig